packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -29,7 +29,7 @@ type TCurrentList = DetailsOfEachMT5Loginid & {
     enabled: number;
 };
 
-type TCFDRealAccountDisplayProps = {
+export type TCFDRealAccountDisplayProps = {
```

Comment:
```suggestion
type TCFDRealAccountDisplayProps = {
```
@arshad-rao-deriv Prop types of a component are scoped to the component itself, Ideally we don't need to export them. If you need the type somewhere else (test files) you can use `React.ComponentProps<typeof ComponentName>` generic to get the type 🙇🏻

Reply:
Updated.